transcom/mymove
# B 17680 Full / Partial PPM Text Not Showing in Closeout Table

## Summary

This fixes the issue where in the Service Counselor's closeout table, the text 'FULL' or 'PARTIAL' is not showing under the "Full or partial PPM" column.

**NOTE:** With this fix, the text will only be shown for the moves that are newly created. This is because the moves that are already created before this fix was not assigned the value 'partial' or 'full' correctly when they were created.

### How to test

1. Create a new move.
2. Add HHG and PPM shipments for a Partial PPM move. Add only PPM shipments for a Full PPM move.
3. Log in as a Counselor in the right GBLOC and finish counseling for the move.
4. Log in as a Counselor in the Closeout location GBLOC and switch to the "PPM Closeout" tab.
5. You should be able to see "Full" or "Partial" under "Full or partial PPM" column.

### Backend
